modules/installation-configuration-parameters.adoc

```diff
@@ -246,8 +250,14 @@ ifdef::osp[]
 The string must be 14 characters or fewer long.
 endif::osp[]
 
+ifndef::ibm-power-vs[]
 |`platform`
 |The configuration for the specific platform upon which to perform the installation: `alibabacloud`, `aws`, `baremetal`, `azure`, `ibmcloud`, `openstack`, `ovirt`, `vsphere`, or `{}`. For additional information about `platform.<platform>` parameters, consult the table for your specific platform that follows.
+endif::ibm-power-vs[]
```

Comment:
@alishaIBM You don't need conditionals here. Just add `powervs` to line 255. 
Remove lines 253, 256-260

Reply:
okay. thanks @SNiemann15